industrial_ci/util.sh

```diff
@@ -86,15 +86,14 @@ function _end_fold_script {
     color_wrap=${2:-32}
 
     if [ $exit_code -eq "1" ]; then color_wrap=31; fi  # Red color
-    if [ -z $TRAVIS_FOLD_NAME ]; then
+    if ! [ -z $TRAVIS_FOLD_NAME ]; then
```

Comment:
It is intended, but the else block still makes sense.
` if ! [ -z $TRAVIS_FOLD_NAME ]` -> `if $TRAVIS_FOLD_NAME is not empty` -> `if fold is open`
This is the bug I did not find in #100